### ISSUE TITLE
Question on text on TRS

### ISSUE THREAD

**Issue by ppdewolf** [julienjamme/handbook_sdc_from_doc_to_md#75]
Not sure where/how to place these kind of questions:

The second displayed formule includes a union symbol with a dot above it. Is this intended to denote a "disjoint union"? If so, is it actually *needed* to be a "*disjoint* union" symbol? Explain the symbol in the text?

At the end of the text on TRS (chapter 3) it states 
$\mathbf{x}_{1(q)},\ldots,\mathbf{x}_q$
Is this correct? What does the subscript $1(q)$ mean here?

What are all the different x-variable-notations: 
$`\{x\}_{i,j}`$ (subscripts $`i,j`$)
$`\mathbf{x}_{q_1},\ldots ,\mathbf{x}_{q_Q}`$ (subscripts $`q_1`$ to $`q_Q`$)
$`\mathbf{x}_{t_1},\ldots ,\mathbf{x}_{t_T}`$ (subscripts $`t_1`$ to $`t_T`$)

There are $p$ characteristics mention at the start of the text, but also $Q$ and $T$ are mentioned. Please explain what each notation means (if different) or make the notation consistent.

**Comment by JohannesGuss** [comment]
@ppdewolf

The implementation (used by R and muArgus) will always treat the input as "disjoint union", otherwise a units could belong to two or more municipalities/girdcells/.... Or do I missunderstand your question?

The first one  $\{x\}_{i,j} = \mathbf{X} \in \mathbb{R}^{n\times p}$ is a matrix notation.

```math
\mathbf{x}_{q_1},\ldots,\mathbf{x}_{q_Q} \qquad \text{,} \qquad \mathbf{x}_{t_1},\ldots,\mathbf{x}_{t_T}
```
are (possibly different) subsets of variables. Agree that this is not ideal. Tried to fix with PR [#93](https://github.com/julienjamme/handbook_sdc_from_doc_to_md/pull/93)

Edit: I was apparently not able to make github render all the LaTeX in this comment in-line...

**Comment by ppdewolf** [comment]
Accepted
